deflate.c

```diff
@@ -52,7 +52,7 @@
 #include "deflate_p.h"
 #include "functable.h"
 
-const char zng_deflate_copyright[] = " deflate 1.2.11.f Copyright 1995-2016 Jean-loup Gailly and Mark Adler ";
+ZLIB_INTERNAL const char zng_deflate_copyright[] = " deflate 1.2.11.f Copyright 1995-2016 Jean-loup Gailly and Mark Adler ";
```

Comment:
Does this still prevent the compiler from optimizing it out? It should remain in the finished binary.
We should probably add our own string to this.

Should probably say 1.2.12.f now.

Reply:
I have updated the version numbers to 1.2.12.f. I removed `ZLIB_INTERNAL` and added `PREFIX` instead.